bentoml/cli/deployment.py

```diff
@@ -219,9 +219,15 @@ def deployment():
         type=click.INT,
     )
     @click.option(
-        '--api-name',
-        help='User defined API function will be used for inference. Option applicable'
+        '--gunicorn-workers-per-instance',
```

Comment:
let's make this the same as it is in the proto message `num_of_gunicorn_workers_per_instance`? or make them both `gunicorn workers per instance`?

Reply:
I removed `num_of_`.  It sounds weird to me in CLI context.  I will just add it back.